llvm/include/llvm/TargetParser/PPCTargetParser.def

```diff
@@ -84,6 +85,7 @@
 #define AIX_PPC8_VALUE 0x00010000
 #define AIX_PPC9_VALUE 0x00020000
 #define AIX_PPC10_VALUE 0x00040000
+#define AIX_PPC11_VALUE 0x00080000
```

Comment:
Where did you get this value? Is it documented somewhere? From the latest AIX 7.3 (TL2) header, I didn't get the definition. Maybe we should let the OS team define this value first and then compiler can use it.

Reply:
I don't believe there is a definition for P11 yet, this is just what I expect based off the existing values.

Reply:
We've discussed with the AIX OS team, and this is the value in their development builds